strutil/strutil.go

```diff
@@ -111,3 +111,37 @@ func TruncateOutput(data []byte, maxLines, maxBytes int) []byte {
 	}
 	return data
 }
+
+// ParseValueWithUnit parses a value like 500kB and returns the number
+// in byte
+func ParseValueWithUnit(inp string) (int64, error) {
+	unitMultiplier := map[string]int{
+		"B":  1,
+		"kB": 1000,
```

Comment:
Should we support `KB` in case someone just types that out of a habit? (regardless if they meant 1024 or 1000)

Reply:
`KiB`?

Reply:
I meant that lower case and upper case difference is pretty subtle and may be missed by people. For usability we should perhaps ignore case here.

Reply:
Yeah, I was sitting on the fence about it but I think it does make sense to make the users life easier this way. I will make it ignore the case.